modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavaClientCodegen.java

```diff
@@ -288,8 +291,7 @@ public void processOpts() {
         writeOptional(outputFolder, new SupportingFile("manifest.mustache", projectFolder, "AndroidManifest.xml"));
         supportingFiles.add(new SupportingFile("travis.mustache", "", ".travis.yml"));
         supportingFiles.add(new SupportingFile("ApiClient.mustache", invokerFolder, "ApiClient.java"));
-        supportingFiles.add(new SupportingFile("ServerConfiguration.mustache", invokerFolder, "ServerConfiguration.java"));
```

Comment:
this should stay here

Reply:
see https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/Java/ApiClient.mustache#L68-L94

Reply:
I see. I could have sworn it was for all clients when I reviewed your other PR and didn't see how it was jersey2-only. I regenerated samples against this commit and pushed to check compilation, but I'll make this change and push over that.